main.py

```
@@ -36,7 +36,7 @@
                 retry_count = 0
 
             retry_count += 1
-            logger.error(f"Bot crashed with error: {e}")
+            logger.exception(f"Bot crashed!")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)